plugins/DepGraph/crawlChildren.js

```diff
@@ -13,7 +13,7 @@
 var sep = '\x1f'; // separator
 // var MAX_DEPTH = 100;
 
-function crawlChildren(ptype: string, children: Array<string>, nodes: Map<string, Map>, depth: number, graph: Object) {
+function crawlChildren(ptype: string, children: Array<string>, nodes: Map<string, Map<any>>, depth: number, graph: Object) {
```

Comment:
Adding `any` here just to make flow happy is not good. What are the values in `nodes` really? Do we need to define a new type for it?